pkg/volume/emptydir/empty_dir.go

```diff
@@ -450,7 +450,7 @@ func (ed *emptyDir) setupDir(dir string) error {
 		// the thread, clearing the umask, creating the dir, restoring
 		// the umask, and unlocking the thread, we do a chmod to set
 		// the specific bits we need.
-		err := os.Chmod(dir, perm)
+		err := volumeutil.Chmod(dir, perm)
```

Comment:
Wouldn't this need to be done in every spot `Chmod` is being called? Example: https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/atomic_writer.go#L398

Reply:
Sure, we can do it, although I'm not sure what additional effect it will have.

Reply:
True, I am just worried that there are other places where we are not setting the Windows file permissions correctly